app/src/main/java/de/tum/in/tumcampusapp/activities/generic/ActivityForSearching.java

```diff
@@ -92,7 +92,6 @@ public boolean onCreateOptionsMenu(Menu menu) {
         // If activity gets called via Intent with a search query set SearchView accordingly
         if (mQuery != null) {
             mSearchView.setQuery(mQuery, false);
-            mOpenSearch = true;
```

Comment:
Notice I changed the behavior here, i.e. the search textbox won't be opened if there is a predefined query.
Makes more sense imho.